lisa/wa.py

```diff
@@ -645,3 +645,75 @@ def get_stats(self, **kwargs):
             agg_cols=['iteration', 'frame_id'],
             **kwargs,
         )
+
+
+class WASysfsExtractorCollector(WAArtifactCollectorBase):
+    """
+    WA collector for the syfs-extractor augmentation.
+
+    **Example**::
+
+        def pixel6_energy_meter(df):
+            # Keep only CPU's meters
+            df = df[df.value.str.contains('S4M_VDD_CPUCL0|S3M_VDD_CPUCL1|S2M_VDD_CPUCL2')]
+            df[['variable', 'value']] = df.value.str.split(', ', expand=True)
+
+            def _clean_variable(variable):
+                if 'S4M_VDD_CPUCL0' in variable:
+                    return 'little-energy'
+                if 'S3M_VDD_CPUCL1' in variable:
+                    return 'mid-energy'
+                if 'S2M_VDD_CPUCL2' in variable:
+                    return 'big-energy'
+                return ''
+
+            df['variable'] = df['variable'].apply(_clean_variable)
+            df['value'] = df['value'].astype(int)
+            df['unit'] = "bogo-ujoules"
+
+            # Add a total energy variable
+            df = pd.concat([
+                df, pd.DataFrame(data={
+                    'variable': 'total-energy',
+                    'value': [df['value'].sum()]
+                })]
+            )
```

Comment:
nit: the indentation/parenthesing/line return is a bit of a mess:
```
df = pd.concat((
    df,
    pd.DataFrame(data={
        'variable': 'total-energy',
        'value': [df['value'].sum()]
    })
))
```

Also seems to be missing a `unit` column